src/cli/cmds/fleet.toit

```diff
@@ -429,6 +429,74 @@ create-fleet-commands config/Config cache/Cache ui/Ui -> List:
       --run=:: create-reference it config cache ui
   cmd.add create-reference-cmd
 
+  migration-cmd := cli.Command "migration"
+      --aliases=["migrate"]
+      --help="""
+        Migrate a fleet to a new broker.
+
+        - Add a new broker to your configuration. Use 'config broker add ...'.
+        - Start the migration with 'migration start --broker=<broker>'.
+        - Upload a new pod, and roll it out to the fleet with 'fleet roll-out'.
+        - Check the status of the migration with the 'status' command.
```

Comment:
```suggestion
        - Check the status of the migration with the 'migration status' command.
```

Reply:
There is no specific `migration status` command. The normal `status` command is enough.
Changed it to "fleet's 'status' command".